app/graphql/civis_api_schema.rb

```diff
@@ -0,0 +1,12 @@
+GraphQL::Field.accepts_definitions is_public: GraphQL::Define.assign_metadata_key(:is_public)
```

Comment:
is_public method seems to be not added in graphql controller

Reply:
@anbublacky - it is present in the `before action` and processed in another method.